tests/Services/ArtistDisplayServiceTest.php

```diff
@@ -32,13 +32,13 @@ public function testMalformedModel(): void
 
     public function testSuccess(): void
```

Comment:
This test is failing now - only because you've added '?artist=1' into the href. You just need to add that here and it should be fine